commander/src/bootstrapping/templates/lisk-template-ts/generators/plugin_generator.ts

```diff
@@ -46,7 +46,7 @@ export default class PluginGenerator extends Generator {
 		// Check for existing package.json in root directory to use existing info
 		try {
 			// eslint-disable-next-line
-			this._packageJSON = await import(`${this.destinationRoot()}/src/app/package.json`);
+			this._packageJSON = await import(`${this.destinationRoot()}/package.json`);
```

Comment:
This can be shorthanded by: 

```suggestion
			this._packageJSON = await import(this.destinationPath('package.json'));
```